orttraining/orttraining/python/ort_trainer.py

```diff
@@ -311,17 +255,20 @@ def convert_model_loss_fn_to_onnx(model, loss_fn, model_desc, device, inputs, op
 
     # Other export options to use(this is for backward compatibility).
     other_export_options = {}
+    other_export_options['training'] = True
 
     # This option was added after 1.4 release.
     if LooseVersion(torch.__version__) > LooseVersion('1.4.0'):
         other_export_options['enable_onnx_checker'] = False
+    # This option was added after 1.6 release.
+    if LooseVersion(torch.__version__) >= LooseVersion('1.6.0'):
+        other_export_options['training'] = torch.onnx.TrainingMode.TRAINING
 
     torch.onnx._export(model, tuple(sample_inputs), f,
                        input_names=input_names,
                        output_names=output_names,
                        opset_version=opset_version,
                        dynamic_axes=dynamic_axes,
-                       training=True,
```

Comment:
Is there a PR open on pytorch so it will treat training=True as if training=torch.onnx.TrainingMode.TRAINING? I believe current nightly will return evaluation graph despite training=True.

Reply:
training becomes an enum instead of bool in pytorch nightly. True is not a valid input argument in that sense. This PR also updates this parameter in case PyTorch version is >= 1.6